code/api/src/modules/crate/model.js

```diff
@@ -9,9 +10,11 @@ module.exports = function(sequelize, DataTypes) {
       type: DataTypes.TEXT
     }
   })
-
+// Define relationship here as you would in Models in Rails
```

Comment:
This is linking up `crateProducts` to have `Products` and `Subscription` to have  `crate`s.  Because we can't really nest information in sql, this is how they are connected to each other. The `associate` is a syntax from Graphql or Sequelize? that would normally be done through a sql commmand of `link`?

Reply:
They're connected in SQL in our DB, but you also need to define the relationship in a model. This is Sequlize syntax, and in Rails this would executed with ActiveRecord syntax, which is a Ruby/SQL ORM.